### ISSUE TITLE
Nil exception on animations that have no attachments visible on a given time

### ISSUE THREAD

**Issue by fabiopicchi** [jjay/godot-spine-importer#2]
I have a spine JSON that is like this:
```
[...]
   "animations":{
      "attack":{
         "slots":{
            "assets/idle":{
               "attachment":[ ]
            },
            "fire":{
               "attachment":[
                  {
                     "time":0,
                     "name":null
                  },
                  {
                     "time":0.2333,
                     "name":"assets/fire"
                  },
                  {
                     "time":0.2667,
                     "name":null
                  },
[...]
```

This null name raise an exception at line 537 of importer.gd:
`var visible = key_data["name"].replace("/", "") == attachment_node.get_name()`

**Comment by fabiopicchi** [comment]
Pull request for fix: #3

**Comment by jkb0o** [comment]
Thank you!
